2024/20240816-#35/README.md

```diff
@@ -0,0 +1,24 @@
+# DevOps Kathmandu Meetup: 35th Ed.
+
+## Date and Time
+16th October, 2024 - Friday, 17:00 to 19:00 NPT
+
+## Talks and Speakers
+- "Creating software to Monitor Hardware" - [Sudeep Bhandari](https://www.linkedin.com/in/sudeephb/)
+  - [Slides](....)
```

Comment:
need slide link from **Sudeep Bhandari**

Reply:
@sudeephb Can you please share slide link?

Reply:
Please find the slide link [here](https://docs.google.com/presentation/d/1_4rP00l1VnJiwRGvy2AeNLMsR7Pra_jt04tMsQ1iI9I/edit?usp=sharing).